### PR TITLE
T1 - Added New Component

### PR DESCRIPTION
**Pull request by jainiag2412**

### CODE REVIEW

File: src/app/page.tsx [hunk 1]
```diff
@@ -3,93 +3,13 @@ import styles from './page.module.css'
 
 export default function Home() {
   return (
-    <main className={styles.main}>
```
Comment (umangbp) [inline src/app/page.tsx:6]: Why did you remove this code ?

Comment (jainiag2412) [reply src/app/page.tsx:6]: I removed it because it was not required

File: src/app/page.tsx [hunk 2]
```diff
@@ -3,93 +3,13 @@ import styles from './page.module.css'
...
-          <h2>
-            Deploy <span>-&gt;</span>
-          </h2>
-          <p>
-            Jaini Gandhi
-          </p>
-        </a>
-      </div>
-    </main>
+const Component1 = () => {
```
Comment (umangbp) [inline src/app/page.tsx:11]:
```suggestion
const TitleComponent = () => {
```

**Review by umangbp — CHANGES_REQUESTED** [review]
Please change component name

**Review by umangbp — APPROVED** [review]
